Failed to build webpack2 example

### Steps
Cloned the repository, then ran the following:
npm install
npm run build

### Expected Result
No errors

### Actual Result
Many errors; see attached log file also.
````
PS E:\apps-node\Semantic-UI-React\examples\webpack2> npm run build

> semantic-ui-react-example-webpack2@1.0.0 build E:\apps-node\Semantic-UI-React\examples\webpack2
> webpack

Hash: 8e1be9e74e54b6dc797e
Version: webpack 2.6.0
Time: 281ms
 Asset     Size  Chunks             Chunk Names
app.js  4.54 kB       0  [emitted]  app
   [0] ./src/main.js 1.91 kB {0} [built] [failed] [1 error]

ERROR in ./src/main.js
Module build failed: Error: Plugin 0 specified in "E:\\apps-node\\Semantic-UI-React\\examples\\webpack2\\.babelrc" provided an invalid property of "__wrapped__"
    at Plugin.init (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\plugin.js:131:13)
    at Function.normalisePlugin (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\file\options\option-manager.js:152:12)
    at E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\file\options\option-manager.js:184:30
    at Array.map (native)
    at Function.normalisePlugins (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\file\options\option-manager.js:158:20)
    at OptionManager.mergeOptions (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\file\options\option-manager.js:234:36)
    at OptionManager.init (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\file\options\option-manager.js:368:12)
    at File.initOptions (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\file\index.js:212:65)
    at new File (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\file\index.js:135:24)
    at Pipeline.transform (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-core\lib\transformation\pipeline.js:46:16)
    at transpile (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-loader\lib\index.js:46:20)
    at Object.module.exports (E:\apps-node\Semantic-UI-React\examples\webpack2\node_modules\babel-loader\lib\index.js:163:20)

npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! semantic-ui-react-example-webpack2@1.0.0 build: `webpack`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the semantic-ui-react-example-webpack2@1.0.0 build script 'webpack'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the semantic-ui-react-example-webpack2 package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs semantic-ui-react-example-webpack2
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls semantic-ui-react-example-webpack2
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\msaro_000\AppData\Roaming\npm-cache\_logs\2017-05-23T14_50_17_089Z-debug.log
PS E:\apps-node\Semantic-UI-React\examples\webpack2>
````
### Version
Windows 10
Node version 7.10.0
npm version 4.2.0


[webpack2 example](https://github.com/Semantic-Org/Semantic-UI-React/tree/master/examples/webpack2)

[2017-05-23T14_50_17_089Z-debug.txt](https://github.com/Semantic-Org/Semantic-UI-React/files/1022586/2017-05-23T14_50_17_089Z-debug.txt)

--- a/examples/webpack1/package.json
+++ b/examples/webpack1/package.json
@@ -4,24 +4,27 @@
   "description": "Get started with Semantic UI React and Webpack 1",
   "main": "index.js",
   "scripts": {
+    "analize": "cross-env ANALIZE_ENV=true npm run build",
+    "analize:production": "cross-env ANALIZE_ENV=true npm run build:production",
     "build": "webpack",
     "build:production": "cross-env NODE_ENV=production npm run build"
   },
   "author": "Alexander Fedyashov <a@fedyashov.com>",
   "license": "MIT",
   "dependencies": {
-    "babel-core": "^6.24.0",
-    "babel-loader": "^6.4.0",
-    "babel-preset-es2015": "^6.24.0",
-    "babel-preset-react": "^6.23.0",
-    "babel-preset-stage-1": "^6.22.0",
-    "cross-env": "^3.2.3",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
-    "semantic-ui-react": "^0.68.3",
-    "webpack": "^1.12.14"
+    "babel-core": "^6.24.1",
+    "babel-loader": "^6.4.1",
+    "babel-plugin-lodash": "^3.2.11",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-1": "^6.24.1",
+    "cross-env": "^5.0.0",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "semantic-ui-react": "^0.68.4",
+    "webpack": "^1.15.0"
   },
   "devDependencies": {
-    "webpack-bundle-analyzer": "^2.3.1"
+    "webpack-bundle-analyzer": "^2.8.2"
   }
 }

--- a/examples/webpack1/package.json
+++ b/examples/webpack1/package.json
@@ -4,8 +4,8 @@
   "description": "Get started with Semantic UI React and Webpack 1",
   "main": "index.js",
   "scripts": {
-    "analize": "cross-env ANALIZE_ENV=true npm run build",
-    "analize:production": "cross-env ANALIZE_ENV=true npm run build:production",
+    "analyze": "cross-env ANALYZE_ENV=true npm run build",
+    "analyze:production": "cross-env ANALYZE_ENV=true npm run build:production",
     "build": "webpack",
     "build:production": "cross-env NODE_ENV=production npm run build"
   },

--- a/examples/webpack1/webpack.config.js
+++ b/examples/webpack1/webpack.config.js
@@ -2,6 +2,7 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 const path = require('path')
 const webpack = require('webpack')
 
+const analize = !!process.env.ANALIZE_ENV
 const env = process.env.NODE_ENV || 'development'
 
 const webpackConfig = {
@@ -26,7 +27,6 @@ const webpackConfig = {
         NODE_ENV: JSON.stringify(env),
       },
     }),
-    new BundleAnalyzerPlugin(),
   ],
 
   output: {
@@ -39,6 +39,10 @@ const webpackConfig = {
     root: path.resolve('src'),
     extensions: ['', '.js', '.jsx'],
   },
+}
+
+if (analize) {
+  webpackConfig.plugins.push(new BundleAnalyzerPlugin())
 }
 
 if (env === 'production') {

--- a/examples/webpack1/webpack.config.js
+++ b/examples/webpack1/webpack.config.js
@@ -2,7 +2,7 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 const path = require('path')
 const webpack = require('webpack')
 
-const analize = !!process.env.ANALIZE_ENV
+const analyze = !!process.env.ANALYZE_ENV
 const env = process.env.NODE_ENV || 'development'
 
 const webpackConfig = {
@@ -41,7 +41,7 @@ const webpackConfig = {
   },
 }
 
-if (analize) {
+if (analyze) {
   webpackConfig.plugins.push(new BundleAnalyzerPlugin())
 }
 

--- a/examples/webpack2/package.json
+++ b/examples/webpack2/package.json
@@ -4,8 +4,8 @@
   "description": "Get started with Semantic UI React and Webpack 2",
   "main": "index.js",
   "scripts": {
-    "analize": "cross-env ANALIZE_ENV=true npm run build",
-    "analize:production": "cross-env ANALIZE_ENV=true npm run build:production",
+    "analyze": "cross-env ANALYZE_ENV=true npm run build",
+    "analyze:production": "cross-env ANALYZE_ENV=true npm run build:production",
     "build": "webpack",
     "build:production": "cross-env NODE_ENV=production npm run build"
   },

--- a/examples/webpack2/package.json
+++ b/examples/webpack2/package.json
@@ -4,24 +4,27 @@
   "description": "Get started with Semantic UI React and Webpack 2",
   "main": "index.js",
   "scripts": {
+    "analize": "cross-env ANALIZE_ENV=true npm run build",
+    "analize:production": "cross-env ANALIZE_ENV=true npm run build:production",
     "build": "webpack",
     "build:production": "cross-env NODE_ENV=production npm run build"
   },
   "author": "Alexander Fedyashov <a@fedyashov.com>",
   "license": "MIT",
   "dependencies": {
-    "babel-core": "^6.24.0",
-    "babel-loader": "^6.4.0",
-    "babel-preset-es2015": "^6.24.0",
-    "babel-preset-react": "^6.23.0",
-    "babel-preset-stage-1": "^6.22.0",
-    "cross-env": "^3.2.3",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
-    "semantic-ui-react": "^0.68.3",
-    "webpack": "^2.3.0"
+    "babel-core": "^6.24.1",
+    "babel-loader": "^7.0.0",
+    "babel-plugin-lodash": "^3.2.11",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-1": "^6.24.1",
+    "cross-env": "^5.0.0",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "semantic-ui-react": "^0.68.4",
+    "webpack": "^2.6.1"
   },
   "devDependencies": {
-    "webpack-bundle-analyzer": "^2.3.1"
+    "webpack-bundle-analyzer": "^2.8.2"
   }
 }

--- a/examples/webpack2/webpack.config.js
+++ b/examples/webpack2/webpack.config.js
@@ -2,7 +2,7 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 const path = require('path')
 const webpack = require('webpack')
 
-const analize = !!process.env.ANALIZE_ENV
+const analyze = !!process.env.ANALYZE_ENV
 const env = process.env.NODE_ENV || 'development'
 
 const webpackConfig = {
@@ -44,7 +44,7 @@ const webpackConfig = {
   },
 }
 
-if (analize) {
+if (analyze) {
   webpackConfig.plugins.push(new BundleAnalyzerPlugin())
 }
 

--- a/examples/webpack2/webpack.config.js
+++ b/examples/webpack2/webpack.config.js
@@ -2,6 +2,7 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 const path = require('path')
 const webpack = require('webpack')
 
+const analize = !!process.env.ANALIZE_ENV
 const env = process.env.NODE_ENV || 'development'
 
 const webpackConfig = {
@@ -26,7 +27,6 @@ const webpackConfig = {
         NODE_ENV: JSON.stringify(env),
       },
     }),
-    new BundleAnalyzerPlugin(),
   ],
 
   output: {
@@ -42,6 +42,10 @@ const webpackConfig = {
     ],
     extensions: ['.js', '.jsx'],
   },
+}
+
+if (analize) {
+  webpackConfig.plugins.push(new BundleAnalyzerPlugin())
 }
 
 if (env === 'production') {

